charts/aws-fsx-openzfs-csi-driver/values.yaml

```diff
@@ -4,9 +4,9 @@
 
 image:
   # TODO: replace this temporary private ECR and regenerate the manifests when we are ready to release
-  repository: 633339324534.dkr.ecr.us-east-1.amazonaws.com/aws-fsx-openzfs-csi-driver
+  repository: public.ecr.aws/fsx-csi-driver/aws-fsx-openzfs-csi-driver
```

Comment:
_Technically_ we shouldn't be making this change until AFTER the image has been uploaded to the public ECR. But because this is the initial release we're fine here.